api/src/test/java/org/openmrs/api/handler/PatientDataUnvoidHandlerTest.java

```diff
@@ -144,4 +149,39 @@ public void handle_shouldNotUnvoidTheOrdersAndEncountersThatNeverGotVoidedWithTh
 		Assert.assertNotNull(testOrder.getVoidReason());
 		
 	}
+	
+	/**
+	 * @see PatientDataUnvoidHandler#handle(Patient,User,Date,String)
+	 */
+	@Test
+	@Verifies(value = "should unvoid the members associated with the patient", method = "handle(Patient,"
+			+ "User,Date,String)")
```

Comment:
That's not what I meant, what I meant is that if you are testing code in PatientDataUnvoidHandler then the test should be in PatientDataUnvoidHandlerTest